modules/serverless-openshift-routes.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+// * serverless/networking/serverless-configuring-routes.adoc
+
+[id="serverless-openshift-routes_{context}"]
+= Configuring {product-title} routes for Knative services
+
+If you want to configure a Knative service to use your TLS certificate on {product-title}, you must disable the automatic creation of a route for the service by the {ServerlessOperatorName}, and instead manually create a `Route` resource for the service.
+
+.Prerequisites
+
+* The {ServerlessOperatorName} and Knative Serving component must be installed on your {product-title} cluster.
+
+.Procedure
+
+. Create a Knative service that includes the `serving.knative.openshift.io/disableRoute=true` annotation:
++
+.Example YAML
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: <service_name>
+  annotations:
+    serving.knative.openshift.io/disableRoute: true
+spec:
+  template:
+    spec:
+      containers:
+      - image: <image>
+----
++
+.Example `kn` command
+[source,terminal]
+----
+$ kn service create hello-example \
+  --image=gcr.io/knative-samples/helloworld-go \
+  --annotation serving.knative.openshift.io/disableRoute=true
+----
+
+. Verify that no {product-title} route has been created for the service:
++
+.Example command
+[source,terminal]
+----
+$ oc get routes.route.openshift.io -l serving.knative.openshift.io/ingressName=$KSERVICE_NAME -l serving.knative.openshift.io/ingressNamespace=$KSERVICE_NAMESPACE -n knative-serving-ingress
+----
++
+You should see the following output:
++
+[source,terminal]
+----
+No resources found in knative-serving-ingress namespace.
+----
+
+. Create a `Route` object in the `knative-serving-ingress` namespace by copying the following sample YAML and modifying the replaceable values:
++
+[source,yaml]
+----
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  annotations:
+    haproxy.router.openshift.io/timeout: 600s <1>
+  name: <route_name> <2>
+  namespace: knative-serving-ingress <3>
+spec:
+  host: <service_host> <4>
+  port:
+    targetPort: http2
+  to:
+    kind: Service
+    name: kourier
+    weight: 100
+  tls:
+    insecureEdgeTerminationPolicy: Allow
+    termination: edge <5>
+    key: |-
+      -----BEGIN PRIVATE KEY-----
+      [...]
+      -----END PRIVATE KEY-----
+    certificate: |-
+      -----BEGIN CERTIFICATE-----
+      [...]
+      -----END CERTIFICATE-----
+    caCertificate: |-
+      -----BEGIN CERTIFICATE-----
+      [...]
+      -----END CERTIFICATE----
+  wildcardPolicy: None
+----
+<1> The timeout value for the {product-title} route. You should set the same value as the `max-revision-timeout-seconds` setting (`600s` by default).
```

Comment:
Is there a way to remove `should` from this? Is `must` too strong in this case?

>! should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as
appropriate.